lib/util/regionList.ts

```diff
@@ -22,16 +22,31 @@ export class RegionList {
 
   public getBuffer(): PolusBuffer {
     let buffer = new PolusBuffer();
-    buffer.write32(this.selected); // selected server
+    buffer.write32(this.selected, false); // selected server
     buffer.writeString(this.name); // region name
     buffer.writeString(this.address); // region ping ip
-    buffer.write32(this.servers.length); // server count
+    buffer.write32(this.servers.length, false); // server count
     for (let server of this.servers) {
       buffer.writeString(server.name); //server name: usually <region-name>-MASTER-<number>
       buffer.writeBytes(server.address.split(".").map((e) => parseInt(e))); // server ip as a four byte array
-      buffer.write16(server.port); // server port
-      buffer.write32(server.failures); // number of connection failures
+      buffer.write16(server.port, false); // server port
+      buffer.write32(server.failures, false); // number of connection failures
     }
     return buffer;
   }
+
+  public fromBuffer(buffer: PolusBuffer) {
```

Comment:
I think this should be static and return a new instance of `RegionList`. Doing so would allow the removal of the default values in the constructor which could be a bit confusing.

Reply:
fixed, see 0e898ef

Reply:
Perfect, thanks